config/jedi_update_exclude_list.txt

```diff
@@ -0,0 +1,5 @@
+/tmp/
```

Comment:
we probably don't want to put this in /config but in /etc/<somewhere/jedi_update_exclude_list.txt